optimizely/optimizely_user_context.py

```diff
@@ -114,3 +133,147 @@ def as_json(self):
             'user_id': self.user_id,
             'attributes': self.get_user_attributes(),
         }
+
+    def set_forced_decision(self, flag_key, rule_key, variation_key):
```

Comment:
default None for optional rule_key? position? We can consider to split to 2 functions with and without rule_key.